### PR TITLE
Test faucet patch fix

### PR DESCRIPTION
**Pull request by 0xmovses**
# Summary
NB. This is not for merging, just for testing out a patch fix in aptos-core fork. Reviewers are free to test out the commands if they wish. 
This branch was cut from `origin/eng-442/malicious_load_soak_test` and updates the rev tag to bring in the changes from aptos-core PR https://github.com/movementlabsxyz/aptos-core/pull/42

When running these tests without the patch fix, the tests run and then hangs, then eventually timesout with an error.
```
thread '<unnamed>' panicked at networks/suzuka/suzuka-client/src/load_soak_testing/mod.rs:387:13:
attempt to divide by zero
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
  2024-07-10T10:31:37.295159Z  WARN suzuka_client::load_soak_testing: log: "Scenario:1 execution failed because: FaucetClientError { inner: Inner { kind: Unknown, source: Some(Unknown(Ledger on endpoint (http://0.0.0.0:30731/v1/) is more than 60s behind current time, timing out waiting for the transaction. Warning, transaction (b716b204) might still succeed.)) } }"
    at networks/suzuka/suzuka-client/src/load_soak_testing/mod.rs:256

thread '<unnamed>' panicked at networks/suzuka/suzuka-client/src/load_soak_testing/mod.rs:387:13:
attempt to divide by zero
```

When running the tests with the patch, the process completes, exits and I see the output below.

# Changelog
- Updates rev tag
- Contains all changes from `malicious_load_soak_test`

# Testing
1. `CELESTIA_LOG_LEVEL=FATAL nix develop --extra-experimental-features nix-command --extra-experimental-features flakes --command bash  -c "just suzuka-full-node native build.setup.local --keep-tui"`
2. In new terminal `cargo build & DOT_MOVEMENT_PATH=".movement" LOADTEST_NUMBER_SCENARIO=4 LOADTEST_NUMBER_SCENARIO_PER_CLIENT=2 target/debug/basicalicebob`

# Outstanding issues
This error still occurs but I think it is unrelated to this test and the test patch fix is still valid:
```
thread '<unnamed>' panicked at networks/suzuka/suzuka-client/src/load_soak_testing/mod.rs:387:13:
attempt to divide by zero
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

# Full output
```
⋊> ~/m/movement on 7b13f6a7 ⨯ DOT_MOVEMENT_PATH=".movement" LOADTEST_NUMBER_SCENARIO=4 LOADTEST_NUMBER_SCENARIO_PER_CLIENT=2 target/debug/basicalicebob                                                                                                                          11:02:33
  2024-07-10T10:02:39.166932Z  INFO suzuka_client::load_soak_testing: Load and Soak test inited with config:ExecutionConfig { kind: Load { number_scenarios: 4 }, logfile: "log_file.txt", execfile: "test_result.txt", number_scenario_per_client: 2 }
    at networks/suzuka/suzuka-client/src/load_soak_testing/mod.rs:50

  2024-07-10T10:02:39.167039Z  INFO suzuka_client::load_soak_testing: Start test scenario execution.
    at networks/suzuka/suzuka-client/src/load_soak_testing/mod.rs:140

  2024-07-10T10:02:39.173426Z  INFO basicalicebob: Scenario:1
=== Addresses ===
Alice: 0x773ea5f49d4735a5b52d20d615cef5193f80437b1de6a20f2982071f7ec0b771
Bob: 0x3c4222d75d671518bb5a0a620cb4985689fc078635fd0ddae6c3bd0338babcd9
    at networks/suzuka/suzuka-client/bin/basicalicebob.rs:67

  2024-07-10T10:02:39.173426Z  INFO basicalicebob: Scenario:3
=== Addresses ===
Alice: 0x659e2d9fb3a7c35ac815f9f77f02231758c1545735320328c6c9a3b989a31db2
Bob: 0x73da5005625cb380560d64311c8da65f3e8ac865e9cd7e388cc7eae77790227a
    at networks/suzuka/suzuka-client/bin/basicalicebob.rs:67

  2024-07-10T10:02:39.173447Z  INFO basicalicebob: 1 Before alice fund
    at networks/suzuka/suzuka-client/bin/basicalicebob.rs:74

  2024-07-10T10:02:39.173450Z  INFO basicalicebob: 3 Before alice fund
    at networks/suzuka/suzuka-client/bin/basicalicebob.rs:74

  2024-07-10T10:02:39.174536Z  INFO basicalicebob: Scenario:2
=== Addresses ===
Alice: 0xc32c57e062ade673b0a86bb771ba3b7166e63293b90df947a8d4398b47c4a453
Bob: 0x95882b9af9ce61e836feecb14adfa6d4e1d7844f5d337c268a556845f5f6f7a1
    at networks/suzuka/suzuka-client/bin/basicalicebob.rs:67

  2024-07-10T10:02:39.174543Z  INFO basicalicebob: 2 Before alice fund
    at networks/suzuka/suzuka-client/bin/basicalicebob.rs:74

  2024-07-10T10:02:39.174546Z  INFO basicalicebob: Scenario:4
=== Addresses ===
Alice: 0xcbcc990b34033bcd90fd04d9a2dbb8261d07465c168806b5add308497798bb1a
Bob: 0x223ec9e28a17f482a4a0a0b2b165cb63e4f4fb192913adf8273777475f3607eb
    at networks/suzuka/suzuka-client/bin/basicalicebob.rs:67

  2024-07-10T10:02:39.174553Z  INFO basicalicebob: 4 Before alice fund
    at networks/suzuka/suzuka-client/bin/basicalicebob.rs:74

  2024-07-10T10:02:39.176744Z  WARN suzuka_client::load_soak_testing: log: "Scenario:4 execution failed because: FaucetClientError { inner: Inner { kind: HttpStatus(503), source: None } }"
    at networks/suzuka/suzuka-client/src/load_soak_testing/mod.rs:256

  2024-07-10T10:02:39.176776Z  WARN suzuka_client::load_soak_testing: log: "Scenario:2 execution failed because: FaucetClientError { inner: Inner { kind: HttpStatus(503), source: None } }"
    at networks/suzuka/suzuka-client/src/load_soak_testing/mod.rs:256

  2024-07-10T10:02:39.176973Z  WARN suzuka_client::load_soak_testing: log: "Scenario:1 execution failed because: FaucetClientError { inner: Inner { kind: HttpStatus(503), source: None } }"
    at networks/suzuka/suzuka-client/src/load_soak_testing/mod.rs:256

thread '<unnamed>' panicked at networks/suzuka/suzuka-client/src/load_soak_testing/mod.rs:387:13:
attempt to divide by zero
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
  2024-07-10T10:02:41.203310Z  INFO basicalicebob: 3 Before Bod create_account
    at networks/suzuka/suzuka-client/bin/basicalicebob.rs:79

  2024-07-10T10:02:43.227024Z  INFO basicalicebob: 3 After Bod create_account
    at networks/suzuka/suzuka-client/bin/basicalicebob.rs:82

  2024-07-10T10:02:43.234478Z  INFO basicalicebob: Scenario:3
=== Initial Balances ===
Alice: 100000000
Bob: 0
    at networks/suzuka/suzuka-client/bin/basicalicebob.rs:86

  2024-07-10T10:02:45.265277Z  INFO basicalicebob: Scenario:3
=== Intermediate Balances ===
Alice: 99998400
Bob: 1000
    at networks/suzuka/suzuka-client/bin/basicalicebob.rs:110

  2024-07-10T10:02:47.298892Z  INFO basicalicebob: Scenario:3
=== Intermediate Balances ===
 Alice: 99996800
 Bob: 2000
    at networks/suzuka/suzuka-client/bin/basicalicebob.rs:136

  2024-07-10T10:02:49.329504Z  INFO basicalicebob: Scenario:3
=== Final Balances ===
 Alice: 99995200
 Bob: 3000
    at networks/suzuka/suzuka-client/bin/basicalicebob.rs:162
    ```

### CODE REVIEW

**Comment by mzabaluev** [comment]
A lot of accidentally committed files need to be cleaned up under `./networks/suzuka/suzuka-client/bin/malicious/move/.aptos`.

File: log_file.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+[2m2024-07-10T10:02:39.176760Z[0m [33m WARN[0m [2msuzuka_client::load_soak_testing[0m[2m:[0m [3mlog[0m[2m=[0m"Scenario:4 execution failed because: FaucetClientError { inner: Inner { kind: HttpStatus(503), source: None } }"
```
Comment (mzabaluev) [inline log_file.txt:1]: Should this log file be committed?

File: test_result.txt [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{"timestamp":"2024-07-10T10:02:39.173458Z","level":"INFO","fields":{"msg":"3 Before alice fund"},"target":"exec"}
```
Comment (mzabaluev) [inline test_result.txt:1]: Should this results file be committed?

Comment (musitdev) [reply test_result.txt:1]: No remove them. They are created by the load and soak test for future test analysis. I think I'll disable them by default.

**Pull request closed without merging**